websites/S/Scantrad/presence.ts

```diff
@@ -0,0 +1,54 @@
+const presence = new Presence({
+    clientId: "817856303719907389"
+}),
+    browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = { 
+        largeImageKey: "logo",
+        startTimestamp: browsingStamp
+    };
+
+    if (document.location.pathname.includes("/mangas")) {
+        const chapter = document.getElementById("selectCh") as HTMLSelectElement,
+            page = document.getElementById("selectPg") as HTMLSelectElement;
+
+        presenceData.details = "Reading " + document.querySelector(".tl-titre").textContent + ":";
+        presenceData.state = chapter.options[chapter.selectedIndex].text.replace('Chapitre', 'Chapter') + " | " + page.options[page.selectedIndex].text;
+        presenceData.smallImageKey = "reading";
+        presenceData.smallImageText = "Reading a scan";
+        presenceData.startTimestamp = browsingStamp;
```

Comment:
Yeah uh why you add startTimestamp every time when you already have it added to presenceData object?

Reply:
Fixed and done! :)